api-reference/submitfield-component.md

```diff
@@ -0,0 +1,15 @@
+# &lt;SubmitField /&gt;
+
+
+
```

Comment:
Unnecessary newlines?

Reply:
Copy/pasta. This is in other files as well, I think GitBook does that.